docs/user/getting-started.md

```diff
@@ -288,6 +288,8 @@ ADD agentconfig.yaml /etc/flightctl/config.yaml
 
 Note this is a regular `Containerfile` that you're used to from Docker/Podman, with the difference that the base image referenced in the `FROM` directive is bootable. This means you can use standard container build tools and workflows.
 
+If you wish to use the latest development version of the agent please use the `@redhat-et/flightctl-dev` copr repository instead.
```

Comment:
Please run `make fix-spelling` to interactively go through errors or simply capitalize COPR here and add to .spelling.